docs/modules/ROOT/partials/wire-logger.adoc

```diff
@@ -7,58 +7,54 @@ By default, wire logging is disabled.
 To enable it, you must set the logger `{wirelogger}` level to `DEBUG`
 and apply the following configuration:
 
-====
 [source,java,indent=0]
-.{examplesdir}/wiretap/Application.java
+.{examples-dir}/wiretap/Application.java
+[%unbreakable]
 ----
-include::{examplesdir}/wiretap/Application.java[lines=18..35]
+include::{examples-dir}/wiretap/Application.java[lines=18..35]
```

Comment:
Next time when you are preparing a PR and you want to do a renaming, I would love to see this renaming as a separate PR. Such renaming is not helping the reviewer with checking the real changes.

Reply:
I should probably have given descriptions of all the changes in the PR, but there are so many modifications ...

So, please, let me explain here the reason why I changed the variable name:

unfortunately, `examplesdir` is deprecated in Antora, please check [this](https://docs.antora.org/antora/3.0/whats-new/#deprecation-summary), more specifically check the deprecation summary:

> The partialsdir and examplesdir attributes are deprecated. Use the resource ID of the [partial](https://docs.antora.org/antora/3.0/page/include-a-partial/) or [example](https://docs.antora.org/antora/3.0/page/include-an-example/) to reference the resource instead.

now, if the `examplesdir` variable name is kept, here is what we would get (see the title where the full pass is missing):

![Screenshot 2024-03-27 at 09 31 15](https://github.com/reactor/reactor-netty/assets/650758/77028d2a-f05a-4d45-9aa1-be204e8077dc)

so, this is why I had to change the `examplesdir` variable to another one (I used `examples-dir` instead), and then we now get the correct path in the title of the included source code (except that there is still the `examples$` prefix, but for the moment, I could not fix it):

![Screenshot 2024-03-27 at 09 33 51](https://github.com/reactor/reactor-netty/assets/650758/42bbc4af-159b-4b32-bff8-c6a88bba6a2b)

the renamed variable `examples-dir` is defined in the adoc files that are including the partial wire-logger.adoc, for example, see the heading in the tcp-client.adoc:

```
:sourcedir: {core}/src/main/java
:examples-dir: {examples}/src/main/java/reactor/netty/examples/documentation/tcp/client

```